Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Use Python 3.12 as the base image
+FROM python:3.12-slim
+
+# Set the working directory in the container
+WORKDIR /app
+
+# Copy the requirements file into the container
+COPY requirements.txt .
+
+# Install the Python dependencies
+RUN pip install --no-cache-dir -r requirements.txt
+
+# Copy the application code into the container
+COPY ./app .
+
+# Expose the port the app runs on
+EXPOSE 8000
+ENV PYTHONPATH=/
```

Comment:
because working directory is /app in the container, it fails to resolve module from app directory. so, PYTHONPATH should be re adjusted to root level so that the module resolution would succeed.